Cargo.toml

```diff
@@ -1,18 +1,17 @@
 [package]
 name = "svgtypes"
 version = "0.15.2"
-authors = ["Yevhenii Reizner <razrfalcon@gmail.com>"]
-categories = ["parser-implementations"]
-description = "SVG types parser."
-documentation = "https://docs.rs/svgtypes/"
-keywords = ["svg", "parser", "tokenizer"]
 license = "Apache-2.0 OR MIT"
 edition = "2021"
-# Keep in sync with RUST_MIN_VER in .github/workflows/ci.yml
-rust-version = "1.65"
+description = "SVG types parser."
+repository = "https://github.com/linebender/svgtypes"
+keywords = ["svg", "parser", "tokenizer"]
+categories = ["parser-implementations"]
 readme = "README.md"
-repository = "https://github.com/RazrFalcon/svgtypes"
-exclude = ["benches/", "codegen/", "fuzz/"]
+# Keep in sync with RUST_MIN_VER in .github/workflows/ci.yml and with the relevant README.md files.
+# and with the MSRV in the `Unreleased` section of CHANGELOG.md.
```

Comment:
(MSRV is not currently mentioned in the readme.)

Reply:
It will be soon!